rcl/src/rcl/client.c

```diff
@@ -117,6 +117,7 @@ rcl_client_fini(rcl_client_t * client, rcl_node_t * node)
 rcl_client_options_t
 rcl_client_get_default_options()
 {
+  // !!! MAKE SURE THAT CHANGES TO THESE DEFAULTS ARE REFLECTED IN THE HEADER DOC STRING
```

Comment:
I don't these why this comment is necessary. The same applies basically to every line of code. If the developer changes anything he needs to consider upgrading related documentation.